### PR TITLE
Implemented schemas.video.category

### PR DESCRIPTION
**Pull request by BenjaminSchaaf**
Added `schemas.video.category` with spec

### CODE REVIEW

File: lib/youku/v2/schemas.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      def video
+        Video.new(client)
+      end
+
+      class Video < Base
+        BASE_URI = "#{BASE_URI}/video"
+
+        # Public: Get video categories.
+        #
+        # See: http://open.youku.com/docs?id=90
```
Comment (avokhmin) [inline lib/youku/v2/schemas.rb:18]:
```
$ curl -vvv http://open.youku.com/docs?id=90
< HTTP/1.1 301 Moved Permanently
...
< Location: http://cloud.youku.com?id=90
```

Can you change to `http://cloud.youku.com/docs?id=90`

**Comment by avokhmin** [comment]
@BenjaminSchaaf can you update `README.md` and `CHANGELOG.md` too?

**Comment by BenjaminSchaaf** [comment]
I updated/cleaned up the readme, but there is no changelog currently. Is it really worth maintaining a changelog when we have `git log`?

**Comment by avokhmin** [comment]
> I updated/cleaned up the readme, but there is no changelog currently. Is it really worth maintaining a changelog when we have git log?

Why not, it is not hard but very useful. See: https://github.com/Shuttlerock/youku_client/releases

**Comment by BenjaminSchaaf** [comment]
Exactly, we already keep descriptions of high level changes in release tags, so why duplicate them in a changelog which can't always be fast forwarded?

**Comment by avokhmin** [comment]
I try to use best practices, see: https://github.com/plataformatec/devise
They have `CHANGELOG.md` and the same info in https://github.com/plataformatec/devise/releases

**Comment by BenjaminSchaaf** [comment]
ok, I've added a Unreleased section with this pull request

**Comment by avokhmin** [comment]
:+1:

**Pull request merged**